ale_linters/gitcommit/gitlint.vim

```diff
@@ -0,0 +1,50 @@
+" Author: Nick Yamane <nick.diego@gmail.com>
+" Description: gitlint for git commit message files
+
+let g:ale_gitcommit_gitlint_executable =
+\   get(g:, 'ale_gitcommit_gitlint_executable', 'gitlint')
+let g:ale_gitcommit_gitlint_options = get(g:, 'ale_gitcommit_gitlint_options', '')
+let g:ale_gitcommit_gitlint_use_global = get(g:, 'ale_gitcommit_gitlint_use_global', 0)
```

Comment:
This flag isn't used. You should either remove it from here and the documentation, or use it to find the path to the executable in virtualenv with the python function. You might want to do the latter.

Reply:
Oops, done.